src/views/Me.vue

```diff
@@ -228,15 +228,18 @@ const editField = (field: string) => {
                   <a
                     :class="{ [`bg-red-600`]: inscriptions.unpaid[inscription[1].team.id], [`bg-green-600`]: !inscriptions.unpaid[inscription[1].team.id] }"
                     class="center rounded p-2 font-bold text-white transition duration-150 ease-in-out hover:ring hover:ring-pink-500"
-                    href=""
+                    :href="`/tournament/${inscription[1].team.tournament.id }?s=teams`"
                   >
                     {{ inscriptions.unpaid[inscription[1].team.id] ? 'Terminer l\'inscription' : (parseInt(inscription[1].team.players[0]) === user.id || inscription[0] === "manager") ? 'Gérer l\'équipe' : 'Voir l\'équipe' }}
```

Comment:
There still are Typescript errors here

Reply:
There are still 2 Typescript errors due to wrong model changes in the tournament branch. 
This was introduced and reported in the https://github.com/InsaLan/frontend-insalan.fr/pull/54#discussion_r1401652889 PR and will be fixed there.